lib/travis/yaml.rb

```diff
@@ -22,32 +22,32 @@ module Yaml
     # these are meant as examples. we might want to determine the translation
     # in the client
     MSGS = {
-      alert:           'using a plain string as a secure',
+      alert:           'this string should probably be encrypted',
```

Comment:
How do we tell them which string? I'm guessing we don't have a variable due to potential length of the string